apps/total-typescript/src/pages/api/inngest.ts

```diff
@@ -0,0 +1,6 @@
+import {serve} from 'inngest/next'
+import {convertkitSurveyAnswered} from '@skillrecordings/inngest'
+
+export default serve(process.env.NEXT_PUBLIC_SITE_TITLE, [
+  convertkitSurveyAnswered,
+])
```

Comment:
inngest operates as an api endpoint within your app, so as events occur, they trigger configured functions. in this case only the `convertkitSurveyAnswered` function is present